yas-libs/Makefile.am

```diff
@@ -0,0 +1,3 @@
+noinst_LIBRARIES = libyascreen.a
```

Comment:
Using this file in this directory means that no files must be added to the yascreen directory or repo.